models/public/wavernn/description/fatchord_version_fused.py

```diff
@@ -0,0 +1,495 @@
+import torch
```

Comment:
This is clearly a modified version of `fatchord_version` from the original repo. It should be distributed as a patch file instead.

Reply:
Could I also add wavernn_to_onnx.py to this patch file?

Reply:
I suppose you could, but I don't see why you would. It would just make the script harder to read.